src/copyFile.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+
+const copyFile = async(sourcePath, destPath) => {
+  try {
+    const data = await fs.promises.readFile(sourcePath, 'utf-8');
+
+    await fs.promises.writeFile(destPath, data, 'utf-8');
```

Comment:
You should check if the destination file already exists before writing to it. The current implementation will overwrite any existing files without warning.